src/nih_wayfinding/app/scripts/profiles/profile-service.js

```diff
@@ -9,12 +12,49 @@
         var module = {
             getProfile: getProfile,
             getProfileNames: getProfileNames,
-            createProfile: createProfile
+            createProfile: createProfile,
+            setCurrentUser: setCurrentUser,
+            getCurrentUser: getCurrentUser,
+            setUserProperty: setUserProperty
         };
 
         return module;
 
         /**
+         * Get the currently active user.
+         *
+         * @returns Profile object for selected user
+         */
+        function getCurrentUser() {
+            return currentUser;
+        }
+
+        /**
+         * Changes the currently active user.
+         *
+         * @param name {String} User name to set
+         */
+        function setCurrentUser(name) {
+            var user = getProfile(name);
+            if (user) {
+                currentUser = user;
+            } else {
+                console.error('User ' + name + ' does not exist!  Not setting current user.');
+            }
+        }
+
+        /**
+         * Sets a property on the currently active user and saves it to local storage.
+         *
+         * @param property {String} Name of the property to set
+         * @param value {Object} Value to set the property to
+         */
+         function setUserProperty(property, value) {
```

Comment:
Might be better named as 'setCurrentUserProperty'. When I first saw the function signature I assumed I could set a property for any user.